test_scripts/Defects/6_1/3160_SDL_Kill_streaming_thread_by_timeout.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@TMyza Unfortunately current ATF implementation won't allow us to test the issue.
We need implementation of [0125-atf-videostreaming-full-support](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0125-atf-videostreaming-full-support.md) proposal.
Suggested approach with `OnAudio/Video...` notifications is an indirect one and won't work since there is no changes expected in SDL on this notifications.
They should work as before - react on data sent from Mobile to SDL.